v4/source/users.yaml

```diff
@@ -4522,3 +4522,338 @@
               Client.Login("sysadmin@domain.com", "Password1")
     
               ok, response := Client.MigrateAuthToSaml(fromAuthService, usersMap, auto)
+
+  "/users/{user_id}/threads":
+   get:
+    tags:
+     - threads
+    summary: Get all threads that user is following
+    description: |
+      Get all threads that user is following
+
+      __Minimum server version__: 5.29
+
+      ##### Permissions
+      Must be logged in as the user who created the upload sessions.
```

Comment:
I think this should be has permission to edit other user (PERMISSION_EDIT_OTHER_USERS)?

Reply:
@ashishbhate , thanks will update!